.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Beacon",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["beacon"],
```

Comment:
Running a beacon node without connected EL and no checkpoint sync on mainnet will barely do anything, what are you debugging with this?

Reply:
Users can always adapt to their needs, will clarify in docs.
Do you recommend better default?

Reply:
There's also the possibility of loading ENV variable files that could be leveraged.

Reply:
I am not sure yet myself what would be useful to include here but if we wanna include vs code launch configs I would prefer to have only configs that work out of the box without requiring local modification, or otherwise not have them committed at all. The main reason for this is that I have a bunch of configs locally, e.g. the one to run current open test file I shared previously, and tracking `.vscode/launch.json` via git would be annoying for me as my current local launch config doesn't match what this PR introduces.

So if the configs we add via git are good enough so I can delete all the configs I already have locally I am against including anything at all.

Alternative could be to add those as examples in docs, and if someone is using vs code and wants debug configs they  can take the boilerplate from there and modify them locally as needed (not tracked via git)

Reply:
Makes sense. An alternative could be to store them with an alternate name, e.g. `launch.template.json`, and provide a `README.md` detailing what to do. See e.g. [this](https://github.com/microsoft/TypeScript/blob/main/.vscode/launch.template.json)